app/controllers/orgs/google_classroom_configurations_controller.rb

```diff
@@ -29,6 +29,15 @@ def search
 
     def index
       @google_classroom_courses = fetch_all_google_classrooms
+    rescue Google::Apis::AuthorizationError
+      google_classroom_client = GitHubClassroom.google_classroom_client
+      login_hint = current_user.github_user.login
+      redirect_to google_classroom_client.get_authorization_url(login_hint: login_hint, request: request)
+      nil
```

Comment:
Do we need this nil here? I think this was there before to return nil from a helper method

Reply:
Ahh good catch. Yea we don't need this. I'll remove it.